src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static Random randomNumber = new Random();
+    private static ColorSupplier randomColorSupplier = new ColorSupplier();
+
+    public static Ball getRandomBall() {
```

Comment:
Please do not use static methods where possible to avoid it. It's better to create class instance and use it to call the method